src/app/Scheduler/Dockerfile

```diff
@@ -0,0 +1,21 @@
+﻿FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+WORKDIR /app
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src
+COPY ["app/Scheduler/Scheduler.csproj", "app/Scheduler/"]
+COPY ["app/Common.Services/Common.Services.csproj", "app/Common.Services/"]
+COPY ["app/Api.Views/Api.Views.csproj", "app/Api.Views/"]
```

Comment:
Doesn't seem very cool that we need to import `Api.Views` into `Scheduler`. But have no other ideas for now and I guess that is out of scope of this PR. So just highlighting - no actions needed currently.